go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/phil-inc/pcommon
 
-go 1.18
+go 1.22.0
+
+toolchain go1.22.10
```

Comment:
is this ok? won't break any downstream consumers? :thinking:

Reply:
I'm not really sure. Is there an easier way to check than just merging and seeing what breaks?

Reply:
I am okay w/ merging and seeing what breaks as long as we actually track those packages

One thing we could do if we were really afraid is create a feature branch in capi & replace those packages w/ this branch (there is a way with go packages to point at a specific branch in the repo instead of defaulting to HEAD).

I am happy either way. The testing option seems pretty busy to me.

Reply:
Repos that use pcommon:
- capi - util, koanf, ndc, network
- stash - koanf, s3
- hcp-portal - koanf
- data-svc - util, koanf
- thanos - util, koanf, s3, network
- sso - util, koanf, network, postgres
- phil-coupon - util, koanf, network, postgres, db, claim
- phil-enroll - koanf, s3, network